README.md

```diff
@@ -143,4 +156,4 @@ that'll happen when you execute it is a harmless syntax error.
 
 1. Add support for a `--version` flag to `bootstrap-gruntwork-installer.sh` and `gruntwork-install`.
 1. Configure a CI build to automatically set the `--version` flag for each release.
-1. Add automated tests for this repo.
+1. Add an `uninstall` command that uses an `uninstall.sh` script in each module.
```

Comment:
An interesting idea. But do we really need it? Wouldn't we be more likely to build a new AMI / Docker image without the installed script module?